public/app/plugins/datasource/prometheus/querybuilder/components/metrics-modal/state/helpers.ts

```diff
@@ -75,12 +90,10 @@ export function displayedMetrics(state: MetricsModalState, dispatch: React.Dispa
 }
 
 /**
- * Filter the metrics with all the options, fuzzy, type, letter
- * @param metrics
- * @param skipLetterSearch used to show the alphabet letters as clickable before filtering out letters (needs to be refactored)
+ * Filter the metrics with all the options, fuzzy, type, null metadata
  * @returns
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/grafana/grafana/pull/68421/commits/6a9b56a801077a5abf8598a16617dc981c4e1f08